build.gradle

```diff
@@ -13,11 +13,18 @@ buildscript {
     repositories {
         google()
         mavenCentral()
+        maven {
+            url "https://maven.mozilla.org/maven2"
```

Comment:
Even though, we're using nightly maven, we still need "release" maven because Glean depends on it.

Reply:
This would be a great comment right above this line ;-)